### PR TITLE
Release scripts

### PR DESCRIPTION
**Pull request by djosborne**

### CODE REVIEW

File: release-scripts/validate.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+"""
+import re
+
+from docopt import docopt
+
+import utils
+
+if __name__ == "__main__":
+    arguments = docopt(__doc__)
+    utils.validate_markdown_uris()
```
Comment (tomdee) [inline release-scripts/validate.py]: I can't see this function?

File: release-scripts/do_release.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+    (re.compile(r'calico/node-libnetwork:latest'),
+     'calico/node-libnetwork:{libnetwork-version}'),
+
+    (re.compile(r'calico_libnetwork_ver\s*=\s*"latest"'),
+     'calico_libnetwork_ver = "{libnetwork-version}"')
+]
+
+
+# ============== Define the release steps. ===============
```
Comment (tomdee) [inline release-scripts/do_release.py]: I don't get this comment?

File: release-scripts/utils.py [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+                    continue
+            if include:
+                for regex, replace in regex_replace_list:
+                    line = regex.sub(replace, line)
+                new_lines.append(line)
+        assert not master_block, "<!--- start indicator with no end in file %s" % filen
+        replace_file(filen, new_lines)
+
+
+def load_release_data():
```
Comment (tomdee) [inline release-scripts/utils.py]: Get rid of all these unused functions...

**Comment by djosborne** [comment]
Included in #138

**Pull request closed without merging**